process_sarif/sarif_helpers.py

```diff
@@ -88,47 +86,38 @@ def replace_misra_rules(files: List[SarifFile], rules_txt_path: str, verbose=Fal
 
     return files
 
-def find_duplicate_results(results: List[Result], verbose=False) -> List[Result]:
+def remove_duplicate_results(files: List[SarifFile], verbose=False) -> List[SarifFile]:
     '''
-    Takes a list of results, and finds duplicate results across them. The returned
-      Results are the unique Results after removing duplicates.
-    A Result is the same as another if they share an identical Artifact and Region.
-      This is a poor assumption, but a reasonable first pass.
+    Takes a list of results, and removes duplicate results across them. The returned
```

Comment:
Documentation needs a small update to match the new interface

```suggestion
    Takes a list of SARIF files, and removes duplicate results across them. The returned
```